src/Console/UpgradeCommand.php

```diff
@@ -42,9 +42,10 @@ public function handle()
         $this->replacements = [
             'use StoutLogic\\AcfBuilder\\FieldsBuilder;' => 'use Log1x\\AcfComposer\\Builder;',
             'new FieldsBuilder(' => 'Builder::make(',
-            'public function enqueue($block)' => 'public function assets($block)',
-            'public function enqueue($block = [])' => 'public function assets($block)',
-            'public function enqueue()' => 'public function assets($block)',
+            'public function assets($block)' => 'public function assets(array $block)',
```

Comment:
sadly this wont work since PHP will throw an error when ACF Composer gets booted in CLI because the classes don't match the parent

Reply:
I see. How would you like to handle this then?
Add manual update notes like you did for the v3 release?
`sed -i 's/public function assets($block)/public function assets(array $block)/g' app/Blocks/*.php`?

Reply:
> I see. How would you like to handle this then? Add manual update notes like you did for the v3 release? `sed -i 's/public function assets($block)/public function assets(array $block)/g' app/Blocks/*.php`?

yeah sadly it would be another breaking change which i really didn't want to do. i actually intentionally renamed this from `enqueue` to `assets` just to avoid it - but its my bad for not adding the types. :/

i would probably just put it in the upgrade v3 upgrade guide and hope that there has been a lot of upgrades if i were to do this i guess.

Reply:
I also noticed that I missed off the `:void` return type from these upgrades anyway so the end result would be a fatal error.

Reply:
pondering maybe removing `assets()` from the abstract class and doing a `method_exists` check or something instead. not sure, just a thought.

Reply:
Well, I guess `method_exists` makes some sense; not all blocks have assets to enqueue so the user would only add `assets()` when they need it.

Reply:
the other upgrades from `enqueue()` are still missing `:void`